cli/tests/unit/console_test.ts

```diff
@@ -1152,6 +1152,9 @@ Deno.test(function consoleParseCss() {
 });
 
 Deno.test(function consoleCssToAnsi() {
+  const COLORTERM = Deno.env.get("COLORTERM");
+  const TRUE_COLOR = Deno.env.get("CI") || COLORTERM === "truecolor" ||
+    COLORTERM === "24bit";
```

Comment:
Not sure about this. Seems risky to change the behavior of tests at runtime. Would prefer a way to force a specific color mode. Need to think a bit more on how to do that tomorrow.

Reply:
Well, couldn't let it be and rewrote that part today.